README.md

```diff
@@ -5,4 +5,11 @@ Deathstar as a Service
 
 Build a Docker container locally with `make build`.
 
-Build and push to a registry with `make publish`. You can specify the image tag with the env var `IMAGE` (defaults to `cilium/starwars`). 
+Build and push to a registry with `make publish`. You can specify the image tag with the env var `IMAGE` (defaults to `cilium/starwars`)
+
+## Set custom landing response msg
```

Comment:
For clarity in this readme it might be nice to have an H2 something like "Runtime env vars", to distinguish from the "Build and publish" section that mentions setting a build-time env var